server/index.js

```diff
@@ -0,0 +1,31 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var songData = require('../songData');
+var path = require('path');
+var app = express();
+var port = 3000;
+
+app.use(express.static(path.join(__dirname + '../client')));
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+
+// INCLUDE "try" AND "catch" IN THE ASYNC AWAIT BELOW
+
+app.get('/songdata/:id', async (req, res) => {
+  try {
+    var id = req.params.id;
```

Comment:
What in the try will throw an error?

What is are you trying to do that might fail?